helm/fluentd-elasticsearch/values.yaml

```diff
@@ -594,6 +365,12 @@ configMaps:
       logstash_format true
       logstash_prefix "#{ENV['LOGSTASH_PREFIX']}"
       reconnect_on_error true
+      reload_on_failure true
+      reload_connections true
+      log_es_400_reason true
+      template_name "svh_logging"
```

Comment:
The customer part should be added in the configmap on the control plane so we can make this chart reusable. You have to expose this through the values.yaml. Then here for example iterate over the different templates and multiple sources if it is the case

example in corends https://github.com/giantswarm/kubernetes-coredns/blob/master/helm/kubernetes-coredns-chart/templates/configmap.yaml#L30-L36